src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php

```diff
@@ -169,6 +170,20 @@ public function isRequired()
     }
 
     /**
+     * Sets the node as deprecated.
+     *
+     * @param string $message Deprecation message
+     *
+     * @return $this
+     */
+    public function isDeprecated($message = 'The child node "%node%" at path "%path%" is deprecated.')
```

Comment:
should be named `setDeprecated`.
Not sure about the default value, there is always more to add than the default (not a string opinion, just noting.)